### ISSUE TITLE
After I‘ve written something in latex, I can‘t edit it anymore

### ISSUE THREAD

**Issue by 8888Daniel8888** [remnoteio/remnote-issues#173]
**Describe the bug**
When I want to make changes to something I've written in latex, it won't let me
**To Reproduce**
1. write something in latex
2. exit from latex
3. click on it again and try to make changes like adding text

**Expected behavior**
I should be able to continue editing and not write everything all over again
**Screenshots**
![image](https://user-images.githubusercontent.com/69430941/95674118-6adcc380-0bae-11eb-8cb0-39074f359d6a.gif)

**Smartphone (please complete the following information):**
 - Device: iPad Pro 9.7
 - OS: iPadOS 14.0.1
 - Browser safari
 - Version 14

**Additional context**

**Comment by colorizer** [comment]
Yes, I have the same issue too. This happens when I try to append something to the end of the LaTeX code.

**Comment by petermllrr** [comment]
Can confirm on Safari/iPadOS

**Comment by mfranzs** [comment]
Thanks for the report. We've reproduced this and have deployed a fix. Please let us know if you're still having any problems.
